### ISSUE TITLE
Unsupported layer found! Skipping...

### ISSUE THREAD

**Issue by gregspangenberg** [scientific-computing/FKB#11]
Hi thanks for developing this fantastic interface between Fortran and Keras. I have been running into this error however when running the example files:

**../FKB/KerasWeightsProcessing/convert_weights.py:208: UserWarning: Unsupported layer found! Skipping...**

I was hoping that you might have some insight into what the underlying cause of this issue might be and what I could do to address it.
Thank you.

**Comment by jordanott** [comment]
Can you post your keras model construction? For example:
```python
model = Sequential()
model.add(Dense(8, input_dim=5, activation='relu'))
model.add(Dense(12, activation='relu'))
model.add(Dense(2,  activation='linear'))

model.compile(loss='mse', optimizer='SGD')
```
The supported layer types are enumerated [here](https://github.com/scientific-computing/FKB/blob/master/KerasWeightsProcessing/convert_weights.py#L18). If it's not an obvious conflict it probably has to do with the input layer.

**Comment by gregspangenberg** [comment]
Hi thanks for the response!

Using the mnist_keras.py example with the following structure

`model = Sequential()
model.add(Dense(512, activation='relu', input_shape=(784,)))
model.add(Dense(512, activation='relu'))
model.add(Dense(num_classes, activation='softmax'))
model.compile(loss='categorical_crossentropy',
              optimizer=RMSprop(),
              metrics=['accuracy'])`

Dense is listed as one of the supported layer types which is where I am getting lost

**Comment by jordanott** [comment]
I get two warnings when I run that example:
```
../convert_weights.py:209: UserWarning: Unsupported layer found! Skipping...
../convert_weights.py:234: UserWarning: Unsupported activation found! Replacing with Linear
```
The first is from the `InputLayer`, which `convert_weights` handles (so you can ignore it), and the second is from the `softmax` in the output layer. Softmax is not currently implemented on the Fortran side. 

I've [updated](https://github.com/scientific-computing/FKB/commit/8fd8e7d599b3afdf53708d7cc8453b7ff641fcba) the warning message to provide more specific information

**Comment by gregspangenberg** [comment]
Thank you for your help!
